api_test.go

```diff
@@ -538,3 +538,22 @@ func TestAPIAuthOk(t *testing.T) {
 		t.Error("Access to API should have been blocked, but response code was: ", recorder.Code)
 	}
 }
+func TestGetOAuthClients(t *testing.T) {
+	var testAPIID = "1"
+	var responseCode int
+
+	var tempSpecRegister = make(map[string]*APISpec)
+	ApiSpecRegister = &tempSpecRegister
```

Comment:
Since ApiSpecRegister is a global object 😞 you need to clean up for yourself and the end of the test. In your case it probably will be ApiSpecRegister = nil